sys/arm64/include/pcb.h

```diff
@@ -40,8 +40,8 @@ struct pcb {
 	uintptr_t	pcb_x[30];
```

Comment:
This also is way bigger than it needs to be, you only need storage for the saved registers

Reply:
That adds a bit more suffering in kgdb as kgdb has to learn how to tell the two ABIs apart when parsing registers.  I can perhaps do that by checking the embedded value of __FreeBSD_version in the kernel so long as that is bumped at the same time.  Granted, I probably have to do that anyway since fewer registers are saved.